docs/src/architectures/pet.rst

```diff
@@ -8,9 +8,94 @@ PET
+
+A good starting point is to select a cutoff radius that ensures about 20 \
+neighbors on average. This can be done by analyzing the neighbor lists for \
+different cutoffs before launching the training script. `This \
+<https://wiki.fysik.dtu.dk/ase/ase/neighborlist.html>`_ is an example of a \
+neighbor list constructor in Python.
+
+For finite configurations, such as small molecules in COLL/QM9/rmd17 datasets, \
+it makes sense to select R_CUT large enough to encompass the whole molecule. \
+For instance, it can be set to 100 Å, as there are no numerical instabilities \
+for arbitrarily large cutoffs.
+
+The hyperparameter for the cutoff radius is called "R_CUT."
+
+Selection of fitting duration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The second most important group of settings is the one that adjusts the fitting duration \
+of the model. Unlike specifying a dataset-specific cutoff radius, this step is \
+optional since reasonable results can be obtained with the default fitting \
+duration. The time required to fit the model is a complex function of the \
+model's size, the dataset's size, and the complexity of the studied \
+interatomic interactions. The default value might be insufficient for \
+large datasets. If the model is still underfit after the predefined number of \
+epochs, the fitting procedure can be continued by relaunching the fitting \
+script with the same calculation name.
+
+However, the total number of epochs is only part of the equation. The other \
+key aspect is the rate at which the learning rate decays. We use `StepLR \
+<https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.StepLR.html>`_ \
+as a learning rate scheduler. To increase the overall fitting time, one needs \
+to specify a larger step size, which controls how fast StepLR decreases the \
+learning rate. This can be done by specifying the hyperparameter \
+"SCHEDULER_STEP_SIZE."
+
+For hyperparameters like "SCHEDULER_STEP_SIZE," "EPOCH_NUM," "BATCH_SIZE," and \
+"EPOCHS_WARMUP," either normal or atomic versions can be specified. Atomic \
+versions are termed "SCHEDULER_STEP_SIZE_ATOMIC," "EPOCH_NUM_ATOMIC," \
+"BATCH_SIZE_ATOMIC," and "EPOCHS_WARMUP_ATOMIC." For instance, using the same \
+batch size for datasets with structures of very different sizes makes no \
+sense. If one dataset contains molecules with 10 atoms on average and another \
+contains nanoparticles with 1000 atoms, it makes sense to use a 100 times \
+larger batch size in the first case. If "BATCH_SIZE_ATOMIC" is specified, the \
+normal batch size is computed as BATCH_SIZE = BATCH_SIZE_ATOMIC / \
+(average_number_of_atoms_in_the_training_dataset). Similar logic applies to \
+"SCHEDULER_STEP_SIZE," "EPOCH_NUM," and "EPOCHS_WARMUP." In these cases, \
+normal versions are obtained by division by the total number of atoms of \
+structures in the training dataset. All default values are given by atomic \
+versions for better transferability across various datasets.
```

Comment:
The meaning of each of these hyperparameters should be specified, either here or with a link to PET documentation.

Reply:
I just want to say the same. IMO, specifying the of hypers is very important for general understanding. Perhaps it could be done in different sections, like “important” (r_cut, embedding dimensionality, number of gnn layers, etc. ) and “optional” hypers (the rest).

Reply:
@Luthaf, I have made adjustments to all your comments. That is the format, and defining all the mentioned hyperparameters. Please take a look at the current version. 

@abmazitov In the current version I define all the hyperparameters I mention in the practical fitting guide. It is already quite lengthy, thus I believe that it is probably not the place (and not the time) to discuss and define all the other hyperparaeters, such as the ones you mentioned. (Open to discussion).

Reply:
regarding all the other parameters: I don't think they have to be defined **here**, but they should be defined **somewhere**. Do you have a documentation for everything in the PET repo? Then maybe we should link to this documentation. Something like "for more information on the other parameters, please see https://...".

Reply:
I put some description of the hypers. I indeed had some. Please take a look.